apps/.flowconfig

```diff
@@ -0,0 +1,15 @@
+[ignore]
+.*/node_modules/fbjs/.*
+.*/node_modules/react-color/.*
+.*/node_modules/radium/.*
+.*/node_modules/editions/.*
+.*/node_modules/cjson/.*
+<PROJECT_ROOT>/build/.*
+
+[include]
+
+[libs]
+
+[options]
+module.name_mapper='^@cdo/locale$' -> '<PROJECT_ROOT>/src/locale-do-not-import.js'
```

Comment:
Ugh - do we have to maintain this mapping in sync with the ones in webpack.js?  Any way to automate this?